spec/emerald/rubify_spec.rb

```diff
@@ -208,4 +208,67 @@
       end.to raise_error(Emerald::Rubify::InvalidLispFunctionError)
     end
   end
+
+  describe "variable assignment" do
+    it "generates code for global variable assignment of a string" do
+      compiled_code = Emerald::Rubify.new([Emerald::List.new(
+        Emerald::Atom.new("def"),
+        Emerald::Atom.new("input"),
+        Emerald::String.new('"Hello"'))
+                                          ]).rubify
+
+      expect(compiled_code).to eq('input = "Hello"')
+    end
+
+    it "generates code for global variable assignment of a number" do
+      compiled_code = Emerald::Rubify.new([Emerald::List.new(
+        Emerald::Atom.new("def"),
+        Emerald::Atom.new("input"),
+        Emerald::Number.new(4))]).rubify
+
+      expect(compiled_code).to eq("input = 4")
+    end
+
+    it "generates code for global variable assignment of numbers" do
+      compiled_code = Emerald::Rubify.new([Emerald::List.new(
+        Emerald::Atom.new("def"),
+        Emerald::Atom.new("input"),
+        Emerald::List.new(
+          Emerald::Atom.new("+"),
+          Emerald::Number.new(4),
+          Emerald::Number.new(2))
+      )
+                                          ]).rubify
```

Comment:
Oh yeah, tests are more important for sure. I was thinking that linting would be more useful though, because I'm strict with running the tests regularly, but not so much with the linting (as you might have noticed :P)